subiquity/server/controllers/keyboard.py

```diff
@@ -199,7 +200,7 @@ async def set_keyboard(self):
             fp.write(self.model.render_config_file())
         cmds = [
             ['setupcon', '--save', '--force', '--keyboard-only'],
-            ['subiquity-loadkeys'],
+            [resource_path('usr/bin/subiquity-loadkeys')],
```

Comment:
Can you update https://github.com/canonical/subiquity/blob/main/subiquity/models/subiquity.py#L283 as well please?

Reply:
good idea